template-parts/header-personal.php

```diff
@@ -36,11 +36,14 @@
 				</div>
 				<div>
 					<div class="h5 bloginfo-name">
-						<p><?php printf( '<span>%s</span>', esc_attr( get_bloginfo( 'name' ) ) ); ?></p>
-					</div>
-					<div class="mb-0 bloginfo-description">
-						<?php esc_attr( bloginfo( 'description' ) ); ?>
+						<?php printf( '<span>%s</span>', esc_attr( get_bloginfo( 'name' ) ) ); ?>
 					</div>
+					<?php
+						$description = bloginfo( 'description' );
```

Comment:
```suggestion
						$description = get_bloginfo( 'description' );
```
You have to use `get_bloginfo()` if you want to assign the result to a variable. It's some funny behaviour.